### ISSUE TITLE
Error with GO download command

### ISSUE THREAD

**Issue by derekthecool** [gennaro-tedesco/boilit#6]
Hi,

This is a nice project that I would like to use.
I am running into errors when I try to run the `go get` command.

I have no experience with GO.
```
go version
go version go1.10.4 linux/amd64

go get -u -v github.com/gennaro-tedesco/boilit
github.com/gennaro-tedesco/boilit (download)
github.com/spf13/cobra (download)
github.com/spf13/pflag (download)
github.com/spf13/cobra
# github.com/spf13/cobra
go/src/github.com/spf13/cobra/bash_completions.go:22:24: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:383:26: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:425:24: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:437:27: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:475:25: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:486:20: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:501:38: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:529:21: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:566:28: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:591:29: undefined: io.StringWriter
go/src/github.com/spf13/cobra/bash_completions.go:591:29: too many errors
```

**Comment by gennaro-tedesco** [comment]
I tried to reproduce but it works just fine: see below


https://user-images.githubusercontent.com/15387611/126849200-7c167b8f-b1b9-4072-90aa-6bbf73db3ca6.mov

However, given that you are seeing an error while installing the cobra library, you may want to try to install it independently first: see [here](https://github.com/spf13/cobra/issues/204) and [here](https://github.com/spf13/cobra/issues/131) for similar issues. Basically you error you are getting is generated by cobra and not `boilit`. 

Could you please try as indicated in the links and see if it works (also make sure your `GOPATH` is set)?

**Comment by derekthecool** [comment]
@gennaro-tedesco thanks for the prompt reply.
Trying the fixes for cobra install failed for my main computer. Same error as before.
My go path was fine, something must just be strange with my environment.

So I tried it out on a virtual machine running Manjaro instead of my main computer Ubuntu 20.04.
Everything worked fine when I tried it on there. That is good enough for me.
My go path was fine, something must just be strange with my environment.

I am excited to put your awesome neovim boiler plate to use right away.
